pkg/api/validation/dynakube/telemetryservice.go

```diff
@@ -109,3 +115,65 @@ func invalidTelemetryIngestName(_ context.Context, _ *Validator, dk *dynakube.Dy
 func invalidTelemetryIngestNameErrorMessage() string {
 	return fmt.Sprintf(errorTelemetryIngestNoDNS1053Label, validation.DNS1035LabelMaxLength)
 }
+
+func conflictingTelemetryIngestServiceNames(ctx context.Context, dv *Validator, dk *dynakube.DynaKube) string {
+	if !dk.TelemetryIngest().IsEnabled() {
+		return ""
+	}
+
+	dkList := &dynakube.DynaKubeList{}
+	if err := dv.apiReader.List(ctx, dkList, &client.ListOptions{Namespace: dk.Namespace}); err != nil {
+		log.Info("error occurred while listing dynakubes", "err", err.Error())
+
+		return ""
+	}
+
+	dkServiceName := dk.TelemetryIngest().ServiceName
+	if dkServiceName == "" {
+		dkServiceName = dk.TelemetryIngest().GetName()
```

Comment:
i know it was before, but let's rename GetName for GetDefaultServiceName or just GetServiceName 🙏